src/structures/ViewCreateStruct.js

```diff
@@ -14,7 +14,6 @@ define(function () {
      */
     var ViewCreateStruct = function (identifier, type) {
         var query = {
-                "Criteria": {},
```

Comment:
actually we can not do that :( this is BC break if someone wrote something like:

```js
myCreateStruct.body.ContentQuery.Criteria.FullTextCriterion = "whatever"
```